packages/assets-controllers/src/MultichainBalancesController/MultichainBalancesController.ts

```diff
@@ -139,6 +147,8 @@ export class MultichainBalancesController extends BaseController<
   MultichainBalancesControllerState,
   MultichainBalancesControllerMessenger
 > {
+  #isUnlocked = true;
```

Comment:
Just to avoid defaulting to `true`, meaning we expect the constructor to initialize it explicitly.

```suggestion
  #isUnlocked: boolean;
```